packages/uni_app/lib/view/login/login.dart

```diff
@@ -99,37 +97,10 @@ class LoginPageViewState extends State<LoginPageView>
             _loggingIn = false;
           });
         }
-      } catch (err, st) {
+      } catch (err) {
         setState(() {
           _loggingIn = false;
         });
-        if (err is ExpiredCredentialsException) {
-          _updatePasswordDialog();
-        } else if (err is InternetStatusException) {
-          if (mounted) {
-            unawaited(
-              ToastMessage.warning(
-                context,
-                S.of(context).internet_status_exception,
-              ),
-            );
-          }
-        } else if (err is WrongCredentialsException) {
-          if (mounted) {
-            unawaited(
-              ToastMessage.error(
-                context,
-                S.of(context).wrong_credentials_exception,
-              ),
-            );
-          }
-        } else {
-          Logger().e(err, stackTrace: st);
-          unawaited(Sentry.captureException(err, stackTrace: st));
-          if (mounted) {
-            unawaited(ToastMessage.error(context, S.of(context).failed_login));
-          }
-        }
```

Comment:
You need to catch the exception here, but now its called `AuthenticationException` rather than `ExpiredCredentialsException`. Fell free to use the message inside the AuthenticationException object to display to reason to the user by a snack bar or o something else.